web/packages/teleport/src/services/mfa/makeMfa.ts

```diff
@@ -69,7 +69,7 @@ export function parseMfaChallengeJson(
     !challenge.webauthn_challenge &&
     !challenge.totp_challenge
   ) {
-    return null;
+    return;
```

Comment:
Should the return type of this function be `MfaAuthenciateChallenge | undefined`?

We don't have strict null checks enabled right now, but if we did this would not compile as-is.